README.md

```diff
@@ -1,5 +1,9 @@
 # dart_to_js_script_rewriter
 
+[![Pub](https://img.shields.io/pub/v/dart_to_js_script_rewriter.svg)](https://pub.dartlang.org/packages/dart_to_js_script_rewriter)
+[![Build Status](https://travis-ci.org/Workiva/dart_to_js_script_rewriter.svg?branch=travis-ci)](https://travis-ci.org/Workiva/dart_to_js_script_rewriter)
```

Comment:
Doing badges like this, give problems with dartdoc. The link won't work in pub.dartlang.org. Doing it with pure html, will make it work in github as well on pub.